app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,145 @@
  *= require_tree .
  *= require_self
```

Comment:
![image](https://github.com/NelsonAraujo93/RoR-Blog/assets/93039359/77392f35-5209-4eda-9af1-ec04bb01f8ab)

**[OPTIONAL]**

- [ ] It would be nice if you make your commit messages Capitalized to make your pull request professional 👍